libs/category/src/drivers/injection-tokens/category-driver.token.ts

```diff
@@ -1,9 +1,10 @@
 import { InjectionToken } from '@angular/core';
 
+import { DaffProduct } from '@daffodil/product';
+
 import { DaffCategoryServiceInterface } from '../interfaces/category-service.interface';
 
 //TODO(damienwebdev): This import is necessary until we ship Ivy packages, do not remove it.
 //See: https://github.com/ng-packagr/ng-packagr/issues/1844
```

Comment:
this todo needs to move too

Reply:
I think this is going to be removed very soon anyway but changed